### PR TITLE
Updating pointers

### PR DESCRIPTION
**Pull request by LeStarch**
Update pointers to the merged content!

### CODE REVIEW

**Comment by ethancheez** [comment]
should probably fix that ci also

**Comment by LeStarch** [comment]
@ethancheez yup!  I just updated it, will keep shepherding CI!

**Comment by ethancheez** [comment]
Should update [this](https://github.com/fprime-community/fprime-baremetal-reference/blob/main/.github/workflows/baremetal_ci_teensy.yml)
```
 run: |
             mkdir -p ~/.local/bin
             curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | BINDIR=~/.local/bin sh
             cd ~/.local/bin
...
```

To follow these instructions:
```
curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | BINDIR=$VIRTUAL_ENV/bin sh
```

**Comment by LeStarch** [comment]
@ethancheez that is a better fix!

**Comment by LeStarch** [comment]
@ethancheez I pulled that in as I was already touching that file.

File: .github/workflows/baremetal_ci_teensy.yml [hunk 1]
```diff
@@ -34,9 +34,7 @@ jobs:
              pip3 install -r ${{ inputs.fprime_location }}./lib/fprime/requirements.txt
       - name: "Platform Specific Requirements Install"
         run: |
-             mkdir -p ~/.local/bin
-             curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | BINDIR=~/.local/bin sh
-             cd ~/.local/bin
+             curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | BINDIR=$VIRTUAL_ENV/bin sh
              pip3 install arduino-cli-cmake-wrapper==0.2.0a1
              export PATH=~/.local/bin:$PATH
```
Comment (ethancheez) [inline .github/workflows/baremetal_ci_teensy.yml:39]: This line no longer needed since arduino-cli not installed in `~/.local/bin`

Comment (LeStarch) [reply .github/workflows/baremetal_ci_teensy.yml]: Good catch!

Comment (ethancheez) [reply .github/workflows/baremetal_ci_teensy.yml]: beat me to it lol. nice

**Pull request merged**